packages/runtime/lib/runtime.js

```diff
@@ -1148,8 +1148,10 @@ class Runtime extends EventEmitter {
       workerEnv['NODE_OPTIONS'] = `${originalNodeOptions} ${serviceConfig.nodeOptions}`.trim()
     }
 
+    const maxHeapTotal = typeof health.maxHeapTotal === 'string' ? parseMemorySize(health.maxHeapTotal) : health.maxHeapTotal
+
     const maxOldGenerationSizeMb = Math.floor(
-      (health.maxYoungGeneration > 0 ? health.maxHeapTotal - health.maxYoungGeneration : health.maxHeapTotal) / (1024 * 1024)
+      (health.maxYoungGeneration > 0 ? maxHeapTotal - health.maxYoungGeneration : maxHeapTotal) / (1024 * 1024)
```

Comment:
Can you please add support for `parseMemorySize` in `maxYoungGeneration` as well?